### PR TITLE
Settings updates

### PR DESCRIPTION
**Pull request by nicholas-cook**
- Update settings layout to match designs
- Add a way for users to disable head tracking
- Add confirmation dialog to let users know head tracking may not be available outside the app
- Removed the pause button from all screens in the app

### CODE REVIEW

File: app/src/main/java/com/willowtree/vocable/settings/SettingsActivity.kt [hunk 1]
```diff
@@ -21,38 +25,104 @@ class SettingsActivity : BaseActivity() {
...
+    }
+
     override fun getErrorView(): View = error_view
 
     override fun getPointerView(): PointerView = pointer_view
 
     override fun getAllViews(): List<View> {
         if (allViews.isEmpty()) {
-            getAllChildViews(parent_layout)
+            if (settings_confirmation.isVisible) {
```
Comment (MattCorbett488) [inline app/src/main/java/com/willowtree/vocable/settings/SettingsActivity.kt]: Could this be simplified to a `when`?

```
when {
  allViews.isNotEmpty() -> allViews
  settings_confirmation.isVisible -> getAllChildViews(settings_confirmation as ViewGroup)
  else -> getAllChildViews(parent_layout)
}
```

File: app/src/main/java/com/willowtree/vocable/utils/VocableSharedPreferences.kt [hunk 2]
```diff
@@ -38,4 +39,12 @@ class VocableSharedPreferences : KoinComponent {
             encryptedPrefs.edit().putStringSet(KEY_MY_SAYINGS, sayings).apply()
         }
     }
+
+    fun setHeadTrackingEnabled(enabled: Boolean) {
+        encryptedPrefs.edit().putBoolean(KEY_HEAD_TRACKING_ENABLED, enabled).apply()
+    }
+
+    fun getHeadTrackingEnabled(): Boolean {
```
Comment (MattCorbett488) [inline app/src/main/java/com/willowtree/vocable/utils/VocableSharedPreferences.kt]: Could just make this `fun getHeadTrackingEnabled() = encryptedPrefs.getBoolean(KEY_HEAD_TRACKING_ENABLED, true)`?

File: app/src/main/res/layout/vocable_confirmation_dialog.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        tools:text="Leaving the app" />
+
+    <TextView
+        android:id="@+id/dialog_message"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
```
Comment (MattCorbett488) [inline app/src/main/res/layout/vocable_confirmation_dialog.xml]: `dialog_title` and `dialog_message` share a bunch of common attributes - worth extracting to a style or do you see these two TextViews being the only use cases?

File: app/src/main/res/layout/vocable_confirmation_dialog.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        android:layout_height="@dimen/dialog_button_height"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_button_end_margin"
+        android:layout_marginBottom="@dimen/dialog_button_end_margin"
+        android:background="@drawable/vocable_dialog_button_background"
+        android:gravity="center"
+        android:textAllCaps="true"
+        android:textColor="@color/colorPrimary"
+        android:textSize="@dimen/dialog_button_text_size"
+        android:textStyle="bold"
```
Comment (MattCorbett488) [inline app/src/main/res/layout/vocable_confirmation_dialog.xml]: Same deal here with the action buttons and common attributes - worth extracting to a style?

File: app/src/main/res/layout/vocable_confirmation_dialog.xml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/dialog_positive_button"
+        android:layout_width="@dimen/dialog_button_width"
+        android:layout_height="@dimen/dialog_button_height"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_button_end_margin"
+        android:layout_marginBottom="@dimen/dialog_button_end_margin"
+        android:background="@drawable/vocable_dialog_button_background"
+        android:gravity="center"
+        android:textAllCaps="true"
```
Comment (MattCorbett488) [inline app/src/main/res/layout/vocable_confirmation_dialog.xml]: This is true by default I think? (if `ActionButton` is a subclass of `Button`/`AppCompatButton`)

Comment (nicholas-cook) [reply app/src/main/res/layout/vocable_confirmation_dialog.xml]: It's a subclass of `AppCompatButton`, but for some reason the text isn't caps unless that property is in there.

**Pull request merged**